### PR TITLE
Gdpr add

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
merge gdpradd , im doing some change in UI above footer, everything works fine

### CODE REVIEW

File: src/components/WebsiteDemoPage/WebsiteDemoPage.js [hunk 1]
```diff
@@ -4,55 +4,86 @@ import {validateEmail, validatePassword, register, PASSWORD_MAX_LENGTH} from 'se
...
+        };
+    }
+
+    b1StepHandler = (event) => {
+        (event.target.type === "button") ?   [ [$(".row.form-details-1").hide()],[  $(".row.form-details-2").show()] ] : " ";        
+       
+    }
+
+    b2StepHandler = (event) => {
+        (event.target.type === "button") ?   [ [$(".row.form-details-2").hide()],[  $(".row.form-details-3").show()] ] : " ";
```
Comment (ShankyRana) [inline src/components/WebsiteDemoPage/WebsiteDemoPage.js]: try not to use jquery

File: src/components/WebsiteDemoPage/WebsiteDemoPage.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+div {
```
Comment (ShankyRana) [inline src/components/WebsiteDemoPage/WebsiteDemoPage.scss]: contain this scss within a class that is particular to this page only

File: src/components/WebsiteCustStory/WebsiteCustStory.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Link } from "react-router";
+import {validateEmail, validatePassword, register, PASSWORD_MAX_LENGTH} from 'services/FormUtils';
+import { Animated } from "react-animated-css";
+import { Alert, HelpBlock } from 'react-bootstrap';
+import { store } from 'index.js';
+import { toast, style } from 'react-toastify';
+import './WebsiteDemoPage.scss';
+import $ from 'jquery';
```
Comment (ShankyRana) [inline src/components/WebsiteCustStory/WebsiteCustStory.js]: remove jquery

File: src/components/WebsiteCustStory/WebsiteCustStory.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
```
Comment (ShankyRana) [inline src/components/WebsiteCustStory/WebsiteCustStory.js]: remove imports which are not required

File: src/components/WebsiteDemoPage/WebsiteDemoPage.js [hunk 5]
```diff
@@ -4,55 +4,86 @@ import {validateEmail, validatePassword, register, PASSWORD_MAX_LENGTH} from 'se
 import { Animated } from "react-animated-css";
 import { Alert, HelpBlock } from 'react-bootstrap';
 import { store } from 'index.js';
-import { loginSuccess } from 'ducks/auth';
-import { browserHistory } from 'react-router';
-import { toast } from 'react-toastify';
-import { base } from 'services/api';
+import { toast, style } from 'react-toastify';
+import './WebsiteDemoPage.scss';
+import $ from 'jquery';
```
Comment (ShankyRana) [inline src/components/WebsiteDemoPage/WebsiteDemoPage.js]: remove jquery.

File: src/Routes.js [hunk 6]
```diff
@@ -45,7 +45,10 @@ import {
   ConnectPage
 } from 'containers';
 
-import WebsiteDemoPage from './components/WebsiteDemoPage/WebsiteDemoPage';
+import WebsiteDemoPage from 'components/WebsiteDemoPage/WebsiteDemoPage';
```
Comment (ShankyRana) [inline src/Routes.js]: add both of these to components part above

File: src/components/WebsiteCustStory/WebsiteCustStory.js [hunk 7]
```diff
@@ -1,30 +1,203 @@
 import React, { Component } from 'react';
 import { Link } from "react-router";
-import {validateEmail, validatePassword, register, PASSWORD_MAX_LENGTH} from 'services/FormUtils';
+import { validateEmail, validatePassword, register, PASSWORD_MAX_LENGTH } from 'services/FormUtils';
```
Comment (ShankyRana) [inline src/components/WebsiteCustStory/WebsiteCustStory.js]: remove the imports which are not required

File: src/components/WebsiteDemoPage/WebsiteDemoPage.js [hunk 8]
```diff
@@ -32,12 +32,18 @@ class WebsiteDemoPage extends  Component  {
       } 
       
     b1StepHandler = (event) => {
-        (event.target.type === "button") ?   [ [$(".row.form-details-1").hide()],[  $(".row.form-details-2").show()] ] : " ";        
+        (event.target.type === "button") ?   [ [$(".row.form-details-1").fadeOut(),
```
Comment (ShankyRana) [inline src/components/WebsiteDemoPage/WebsiteDemoPage.js]: get rid of jquery , try to use features of react

File: src/components/WebsiteDemoPage/WebsiteDemoPage.scss (hunk 2 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsiteDemoPage/WebsiteDemoPage.scss]: okay sir

**Comment by ShankyRana** [comment]
@GAUTAMRAJU15, do resolve the above conflicts.

File: src/components/WebsiteDemoPage/WebsiteDemoPage.js (hunk 1 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsiteDemoPage/WebsiteDemoPage.js]: should i use plain javascript

File: src/components/WebsiteDemoPage/WebsiteDemoPage.scss (hunk 2 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsiteDemoPage/WebsiteDemoPage.scss]: sir here i have used
.form-details-1
.form-detail-2
.form-detail-3
and is unique to this scss file only . so is this fine sir?

File: src/components/WebsiteCustStory/WebsiteCustStory.js (hunk 4 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsiteCustStory/WebsiteCustStory.js]: removed

**Comment by ShankyRana** [comment]
Use react not plain js.

On Thu, Jun 7, 2018, 00:35 Raju Gautam <notifications@github.com<mailto:notifications@github.com>> wrote:

@GAUTAMRAJU15 commented on this pull request.

________________________________

In src/components/WebsiteDemoPage/WebsiteDemoPage.js<https://github.com/InfluenceIO/frontend/pull/119#discussion_r193525355>:

> +            email : '',
+            Phone :'',
+            company :'',
+            totalemp : '',
+            dept : '',
+            errorEmail: ''
+        };
+    }
+
+    b1StepHandler = (event) => {
+        (event.target.type === "button") ?   [ [$(".row.form-details-1").hide()],[  $(".row.form-details-2").show()] ] : " ";
+
+    }
+
+    b2StepHandler = (event) => {
+        (event.target.type === "button") ?   [ [$(".row.form-details-2").hide()],[  $(".row.form-details-3").show()] ] : " ";


should i use plain javascript

**Pull request closed without merging**